lib/matplotlib/ticker.py

```diff
@@ -2126,6 +2126,8 @@ def get_log_range(lo, hi):
                 a_range = get_log_range(t, -vmin + 1)
             else:
                 a_range = get_log_range(-vmax, -vmin + 1)
+                if vmin<0 and vmax<0:
```

Comment:
@tacaswell 
get_log_range will return a single decade for values bound by two consecutives 2**x:
for instance, -100<x<-10
hence, i'm artificially adding one decade more bounded to 0.